Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN rm android-sdk_r24.4.1-linux.tgz
 ENV ANDROID_COMPONENTS platform-tools,android-23,build-tools-23.0.2
 
 # Install Android tools
-RUN ( sleep 5 && while [ 1 ]; do sleep 1; echo y | /usr/local/android-sdk/tools/android update sdk --filter "${ANDROID_COMPONENTS}" --no-ui -a ; done )
+RUN echo y | /usr/local/android-sdk/tools/android update sdk --filter "${ANDROID_COMPONENTS}" --no-ui -a
```

Comment:
I think it actually has to wait, because it doesn't always return immediately, hence I put a simple loop there in the past.